crates/gitbutler-branch-actions/src/move_commits.rs

```diff
@@ -92,6 +93,22 @@ pub(crate) fn move_commit(
         bail!("the source branch contains hunks locked to the target commit ancestors")
     }
 
+    let mut commits_to_rebase = vec![];
+    let is_head_commit = commit_id == source_branch.head();
+    if !is_head_commit {
+        commits_to_rebase = ctx
+            .repository()
+            .log(source_branch.head(), LogUntil::Commit(commit_id))
+            .unwrap();
+
+        let is_parent_locked =
+            check_source_lock_to_commits(ctx.repository(), &commits_to_rebase, &branch_head_diff);
+
+        if is_parent_locked {
+            bail!("the source branch contains hunks locked to the target commit parents")
```

Comment:
@estib-vega Do you think this error message is correct? Or should it be something like
`the source branch contains hunks locked to the target commit children`?